kevm

```diff
@@ -54,11 +54,18 @@ run_krun() {
     local run_file=$1 ; shift
     run_env "$run_file"
     export K_OPTS=-Xss500m
-    if [[ "$backend" == "haskell" ]]; then
-        krun --directory "$build_dir/$backend/" -cSCHEDULE="$cSCHEDULE" -pSCHEDULE='printf %s' -cMODE="$cMODE" -pMODE='printf %s' "$run_file" "$@" --haskell-backend-command "$build_dir/kore/bin/kore-exec"
-    else
-        krun --directory "$build_dir/$backend/" -cSCHEDULE="$cSCHEDULE" -pSCHEDULE='printf %s' -cMODE="$cMODE" -pMODE='printf %s' "$run_file" "$@"
-    fi
+    case "$backend" in
+        haskell)
+            args=(--haskell-backend-command $build_dir/kore/bin/kore-exec)
+            ;;
```

Comment:
Let's shorten these to one line each:

```
haskel) args=(...) ;;
ocaml)  args=(...) ;;
```

Line up the `a` in `args` and the `;;` for each line.

For the default case can we do `args=()` instead of `args=`? It looks weird.

Reply:
I tried `args=()` and it doesn't work because then `${args[@]}` is an unbound variable. But what I can do instead is `${args[@]-}` so that if `args[@]` is unbound then we use the empty string.